sentinel-adapter/sentinel-spring-webmvc-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/spring/webmvc/SentinelExceptionAware.java

```diff
@@ -0,0 +1,32 @@
+package com.alibaba.csp.sentinel.adapter.spring.webmvc;
```

Comment:
Could you please prepend the license header here?

Reply:
You're right.